packages/frontend/src/components/atoms/Burger.scss

```diff
@@ -0,0 +1,95 @@
+button.navbar-toggler {
```

Comment:
Atoms are not scss file but true component, your file here must go in the AdminLayout.scss or if you want to separate it from it, you can also create a folter in /.../template/AdminLayout/ and put all file related to the AdminLayout in it